### DISCUSSION TITLE
`consult-fd` doesn't work with multiple dirs

### DISCUSSION THREAD

**Discussion by bcardoso** [minad/consult#1052] [Q&A] [answered]
Hello. Thank you for consult!

I'm trying to search multiple dirs with `consult-fd` but it shows no results:

```
(consult-find '("~/test/dir1/" "~/test/dir2/" "~/test/dir3/")) ;; shows results from all dirs as expected
(consult-fd '("~/test/dir1/" "~/test/dir2/" "~/test/dir3/"))  ;; no results
```

From `*consult-async*` buffer:

```
consult--async-process started ("find" "test/dir1/" "test/dir2/" "test/dir3/" "-not" "(" "-path" "*/.[A-Za-z]*" "-prune" ")" "-iregex" ".*aaa.*")
consult--async-process sentinel: event=finished lines=3
consult--async-process started ("fd" "--full-path" "--color=never" "--and" "aaa" "test/dir1/" "test/dir2/" "test/dir3/")
consult--async-process sentinel: event=finished lines=0
```

Am I missing something?

Emacs 29.4, consult 1.7, and fd 10.1.0 on Linux

**Comment by minad** [comment, marked as answer]
This looks like a problem in fd. It also doesn't seem to work on the command line.

EDIT: Works now with https://github.com/minad/consult/commit/8f8cd733c158a2ad9a3bdb9774d04d0f14168975. The fd man page is misleading.

**Reply by bcardoso** [reply]
Thank you!
